stdlib/2and3/argparse.pyi

```diff
@@ -23,6 +23,21 @@ ZERO_OR_MORE = ...  # type: str
 class ArgumentError(Exception): ...
 
 class ArgumentParser:
+    prog = ...  # type: _Text
+    usage = ...  # type: Optional[_Text]
+    description = ...  # type: Optional[_Text]
+    epilog = ...  # type: Optional[_Text]
+    parents = ...  # type: Sequence[ArgumentParser]
```

Comment:
This doesn't exist at runtime.

Reply:
@JelleZijlstra Sorry, but I'm not quite sure what problem you're highlighting. Can you clarify?

Reply:
Mentioned this on Gitter, but I meant that this attribute didn't exist when I constructed an ArgumentParser object in my shell.

Reply:
@JelleZijlstra Sorry for the delay -- I removed the `parents` annotation and rebased the PR.